content/international-candidates.md

```diff
@@ -52,217 +58,144 @@ keywords:
+Alternatively, you may have a pre-existing right to work in the UK, such as settled or pre-settled status under the [EU Settlement Scheme](https://www.gov.uk/settled-status-eu-citizens-families) or [indefinite leave to remain in the UK](https://www.gov.uk/guidance/indefinite-leave-to-remain-in-the-uk).
 
-First, [choose your teaching training provider](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+[If you're an Irish citizen you do not need a visa](https://www.gov.uk/government/publications/common-travel-area-guidance).
 
-Then, contact your chosen teacher training provider to ask about the school where you’ll be employed during your training. The school must be a Home Office licensed employer sponsor – sponsors can help you apply for your visa.
+### Consider getting qualified teacher status
 
-### To work as a teacher
+[You need 'qualified teacher status' (QTS) to teach in many schools in England](https://www.gov.uk/guidance/qualified-teacher-status-qts#overview).
 
-[Search for teaching jobs](https://teaching-vacancies.service.gov.uk/).
+It's not a requirement in all [types of schools](https://www.gov.uk/types-of-school), such as academy schools, free schools, private schools and independent schools. However, having QTS can help your application.
 
-Before you apply for a teaching job, contact the school to check they are a Home Office licensed employer sponsor – sponsors can help you apply for your visa.
+If you’re a qualified teacher from outside the UK you can [work as a teacher in England for up to 4 years without QTS](https://www.gov.uk/guidance/recruit-teachers-from-overseas#employing-overseas-teachers-without-qts-the-4-year-rule).
 
-You’ll be able to apply for your skilled worker visa if:
+[Apply for QTS if you trained outside the UK](https://www.gov.uk/guidance/qualified-teacher-status-qts).
 
-* you have an offer for a salaried training place or a teaching job with a school that is a licensed Home Office employer sponsor
-* you can speak, read, write, and understand English
-* your role pays at least £20,480 or the relevant minimum rate for teachers in the region of England where you are working or training – whichever is higher. If you work part-time, pro-rata rates will apply as long the salary is at least £20,480 per annum
+### Search for a job
 
-[Check minimum rates for teachers by region in England](/salaries-and-benefits).
+<p class="call-to-action__action">
+  <a href="https://teaching-vacancies.service.gov.uk/?utm_source=int_teacher_recruitment&utm_medium=referral&utm_campaign=AY21-22">Search for a teaching <span>job</span></a>
+</p>
 
-## Ways to train or work as a teacher in the UK without a skilled worker visa or a student visa
+### Get support if you're qualified to teach maths, physics or modern languages
 
-### Graduate visa
+If you’re an international qualified teacher of maths, physics, or modern foreign languages at secondary school age, you can get one-to-one support with:
```

Comment:
This doesn't read right to me, 'internationally-qualified' perhaps?

Reply:
I agree w/ Pete